### PR TITLE
Cal-1083 Tasks.update extra fields

### PR DESCRIPTION
**Pull request by Jdww**
### Added
- work_type_id, estimated_duration and assignee_id to tasks.update

Implementation: https://github.com/teamleadercrm/core/pull/8173

## Note
Since we're not ready to release these endpoints yet, we decided to not include the `tasks.apib` in the `src/apiary.apib` file. That way we can iterate on the docs through multiple PRs. Once we're ready to release, we will include `tasks.apib` in `src/apiary.apib`.

### CODE REVIEW

**Review by psrpinto — APPROVED** [review]

**Review by r1dotl — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -3022,6 +3022,9 @@ Update a tasks.
         + id: `00ed6266-a5bd-4aac-a292-2582017b6400` (string, required)
         + description (string, optional)
         + due_at: `2016-02-04T16:44:33+00:00` (string, optional)
+        + work_type_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
+        + estimated_duration: 3600 (number, optional)
```
Comment (ketlane) [inline apiary.apib]: PP: the field `estimated_duration` does not explain the measure. Maybe add a comment here explaining what it's measured in? 

```suggestion
        + estimated_duration: 3600 (number, optional) - In seconds
```

Comment (ketlane) [reply apiary.apib]: Of course, it would need to be added in tasks.apib, not here :D

**Review by psrpinto — DISMISSED** [review]

**Review by ketlane — DISMISSED** [review]

File: apiary.apib (hunk 1 above)
Comment (psrpinto) [reply apiary.apib]: good point 👌

**Review by psrpinto — APPROVED** [review]

**Review by r1dotl — APPROVED** [review]

**Comment by psrpinto** [comment]
@dietrichm @ketlane any chance you could re-approve? Had to merge master.

**Comment by ketlane** [comment]
B: the apiary.apib file does not seem to have the related updates anymore. Maybe run another build?

**Review by ketlane — CHANGES_REQUESTED** [review]
B: the apiary.apib file does not seem to have the related updates anymore. Maybe run another build?

**Comment by psrpinto** [comment]
@ketlane Since we're not ready to release these endpoints yet, we decided to not include the `tasks.apib` in the `src/apiary.apib` file. That way we can iterate on the docs through multiple PRs, otherwise it would have been one gigantic PR for the whole documentation.

We plan on releasing by the end of this sprint, at which point we will include `tasks.apib` in `src/apiary.apib`.

(I'm adding this note to all our open PRs descriptions)

**Comment by ketlane** [comment]
Oh, ok, I understand. Good strategy! Maybe we should do the same with the project items PRs.

**Review by ketlane — APPROVED** [review]

**Pull request merged**